### PR TITLE
Sidebar Room Image Update

### PR DESCRIPTION
**Pull request by mikenrowland**
Updated the sidebar data to display a default room image when no image url is found instead of a # symbol

### CODE REVIEW

**Review by mikenrowland — COMMENTED** [review]
@o4codes @zxenonx @man-of-mind please review

**Review by mikenrowland — COMMENTED** [review]
.

File: backend/utils/sidebar.py [hunk 1]
```diff
@@ -38,7 +38,11 @@ async def __get_room_members(
         for room_member_id in room_members.keys():
             member_data = await DB.get_member(room_member_id, org_members)
             username = member_data.get("user_name", "no user name")
-            image_url = member_data.get("image_url", DEFAULT_DM_IMG)
+            image_url = (
+                DEFAULT_DM_IMG
+                if member_data.get("image_url") == ""
+                else member_data.get("image_url")
+            )
```
Comment (zxenonx) [inline backend/utils/sidebar.py]: Here, we could also use a boolean exp. It's more concise.
```suggestion
            image_url = member_data.get("image_url") or DEFAULT_DM_IMG
```

**Review by zxenonx — COMMENTED** [review]
Thanks, @mykie88.
I've just suggested another way using a boolean expression.

File: backend/utils/sidebar.py (hunk 1 above)
Comment (o4codes) [reply backend/utils/sidebar.py]: Yea
@mykie88 please kindly look at @zxenonx suggestion

**Review by o4codes — CHANGES_REQUESTED** [review]

File: backend/utils/sidebar.py (hunk 1 above)
Comment (mikenrowland) [reply backend/utils/sidebar.py]: Thanks a lot bosses. The suggestion worked like a charm.

**Review by o4codes — APPROVED** [review]
Good work @mykie88

**Review by man-of-mind — APPROVED** [review]

**Review by zxenonx — APPROVED** [review]

**Pull request merged**